Externals/soundtouch/PeakFinder.cpp

```diff
@@ -64,7 +57,7 @@ int PeakFinder::findTop(const float *data, int peakpos) const
 
     refvalue = data[peakpos];
 
-    // seek within �10 points
+    // seek within �10 points

```

Comment:
This character wasn't correctly interpreted by Git. It shows a replacement character.

Reply:
https://gitlab.com/soundtouch/soundtouch/-/blob/master/source/SoundTouch/PeakFinder.cpp
It's the same in the official repository.